config/rbac/net_attach_def_role.yaml

```diff
@@ -0,0 +1,19 @@
+---
+kind: ClusterRole
+metadata:
+  name: net-attach-def-editor
+  labels:
+    app.kubernetes.io/name: role
```

Comment:
Are all those labels really needed?

Reply:
They are nice to have labels helping in manageing and filtering, but not necessary at this point.
Labels removed.

Reply:
I posted PR to remove this labels form other manifest as well https://github.com/AlonaKaplan/selfserviceoverlay/pull/9